guillotina/traversal.py

```diff
@@ -299,6 +299,11 @@ def __init__(self, resource, request, view):
         else:
             request.execute_futures('failure')
 
+        # Add extra headers to response if needed
+        extra_hdrs = getattr(self.view, '__extra_headers__', {})
+        if extra_hdrs:
```

Comment:
It would be great that it can be a function so we could provide a strong_cache() function and generate the headers.